Plots/Contours/NCL_hov_3.py

```diff
@@ -0,0 +1,113 @@
+"""
+NCL_hov_3.py
+================
+This script illustrates the following concepts:
+   - Creating a Hovmueller plot
+   - Hatching fill between contours
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/hov_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/hov_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine
+# and load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/chi200_ud_smooth.nc'))
+
+lon = ds.lon
+times = ds.time
+scale = 1000000
+chi = ds.CHI
+chi = chi/scale
+
+###############################################################################
+# Plot:
+
+# Initialize figure and axis
+fig, ax = plt.subplots(figsize=(7, 7.5))
+
+# Fill area between level 4 contours and level 10 contours with dot hatching
+cf = ax.contourf(lon,
+                 times,
+                 chi,
+                 levels=[4, 10],
+                 colors='None',
+                 hatches=['..'])
+
+# Fill area at the lowest contour level, -6, with line hatching
+cf = ax.contourf(lon,
+                 times,
+                 chi,
+                 levels=[-7, -6],
+                 colors='None',
+                 hatches=['///'])
+
+# Draw contour lines at levels [-6, -4, -2, 0, 2, 4, 6, 8, 10]
+cs = ax.contour(lon,
+                times,
+                chi,
+                levels=np.arange(-6, 12, 2),
+                colors='black',
+                linestyles="-",
+                linewidths=.2)
+
+# Set 0 level contour line to a thicker linewidth
+# cs.levels = [-6, -4, -2, 0, 2, 4, 6, 8, 10]
```

Comment:
Is this comment still needed?

Reply:
It was put in there to demonstrate how to access the levels attribute of a contour set, I tried to clarify it in my comment. Does the new comment make more sense?